compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/lowering_strategy.mlir

```diff
@@ -132,7 +132,7 @@ builtin.module {
 
 // -----
 
-// CHECK-PAD-PACK{LITERAL}: #config = #iree_codegen.lowering_config<tile_sizes = [[128, 128], [0, 0, 128], [64, 64], [0, 0, 8]]>
+// CHECK-PAD-PACK{LITERAL}: #config = #iree_codegen.lowering_config<tile_sizes = [[128, 128], [0, 0, 128], [32, 32], [0, 0, 8]]>
```

Comment:
Not sure why it generates 64 and 128 without this extra step. The original thought of using below codes (16 and 64) is trying to get the relation of 4 times between two tiling levels.
  ```
auto maxL1Size = 16 * scaleFactor;
  auto maxL0Size = 64 * scaleFactor;
```

Reply:
So here scaleFactor = 4. 

What happens is this:

Step 1: 64 is chosen at the AIE core tile size  (maxL1Size is 64)
Step 2: 128 is chosen as memtile size (maxL1Size is 256, but 2432 / 256  = 9.5)
Step 3 then shrinks 64 to 32. 

The logic for steps 1 and 2 was introduced in https://github.com/nod-ai/iree-amd-aie/pull/293 with tests added in https://github.com/nod-ai/iree-amd-aie/pull/299

The above PRs from @nirvedhmeshram  basically changed it from 
Step 1: Choose memtile size
Step 2: Choose AIE core size

To 
Step 1: Choose AIE core size
Step 2: Choose memtile size 

It was an ad hoc solution to getting an important matmul shape to compile. I think what was happening was that for 2432, the memtile size was being chosen as 152 (=19 * 8) which wasn't desirable. Sorry, I can't remember more details than that at the moment

Reply:
Anyway -- I'm happy to go back to the drawingboard for choosing tile sizes. We could consider for example that next gen AIE isn't a 4x4 grid of cores. We should also consider that its the _sum_ of A, B, and C tiles which are constrained simultaneously, so if A is forced to have small tiling due to constraints, the B can have larger tiling.

Reply:
Thanks for your explanation, and I think it now makes sense to me. Also agreed that we should come up with better tiling strategy after recent demos. I'll remove the blocker and you can merge the PR if you like.